src/events/channelCreate.ts

```diff
@@ -0,0 +1,13 @@
+import { Client as DiscordClient, GuildChannel } from "discord.js";
+import ServerSettingsRepository from "../repository/serverSettings";
+import { SetMutedPermissionsForChannel } from "../lib/mutedRole";
+
+export default async function ChannelCreateEvent(discordClient: DiscordClient, channel: GuildChannel) {
```

Comment:
This event does not get called on channelCreate.

Please see `EventBind` in `routes.ts`

Reply:
Weird, this was explicitly tested. I guess it got lost somewhere on the way.